x-pack/plugins/apm/public/components/shared/TransactionActionMenu/TransactionActionMenu.tsx

```diff
@@ -84,92 +55,46 @@ export function TransactionActionMenu({ transaction }: Props) {
     urlParams,
   });
 
-  const closePopover = () => {
-    setIsActionPopoverOpen(false);
-    setIsCustomLinksPopoverOpen(false);
-  };
-
-  const toggleCustomLinkFlyout = () => {
-    closePopover();
-    setIsCustomLinkFlyoutOpen((isOpen) => !isOpen);
-  };
-
-  const toggleCustomLinkPopover = () => {
-    setIsCustomLinksPopoverOpen((isOpen) => !isOpen);
-  };
-
   return (
     <>
-      {isCustomLinkFlyoutOpen && (
-        <CustomLinkFlyout
-          defaults={{ filters }}
-          onClose={toggleCustomLinkFlyout}
-          onSave={() => {
-            toggleCustomLinkFlyout();
-            refetch();
-          }}
-          onDelete={() => {
-            toggleCustomLinkFlyout();
-            refetch();
-          }}
-        />
-      )}
       <ActionMenu
         id="transactionActionMenu"
-        closePopover={closePopover}
+        closePopover={() => setIsActionPopoverOpen(false)}
         isOpen={isActionPopoverOpen}
         anchorPosition="downRight"
         button={
           <ActionMenuButton onClick={() => setIsActionPopoverOpen(true)} />
         }
       >
         <div>
```

Comment:
Maybe we should add `max-height` and `overflow-y: auto` to add a scroll after clicking on `show more`.

<img width="512" alt="Screenshot 2020-11-20 at 11 05 06" src="https://user-images.githubusercontent.com/55978943/99787942-b0cb6680-2b20-11eb-82f3-c36ec9562113.png">

Reply:
Good idea. I'll do that.